core/application/app_configuration.hpp

```diff
@@ -100,11 +101,6 @@ namespace kagome::application {
      */
     virtual log::Level verbosity() const = 0;
 
-    /**
-     * @return true if node in only finalizing mode, otherwise false.
-     */
-    virtual bool isOnlyFinalizing() const = 0;
```

Comment:
Please remove 
```
MOCK_CONST_METHOD0(isOnlyFinalizing, bool());
```
in `/Users/igor/cpp/kagome/test/mock/core/application/app_configuration_mock.hpp` too

Reply:
Thanks